lib/fizzy/instantiate.cpp

```diff
@@ -373,6 +373,16 @@ std::unique_ptr<Instance> instantiate(Module module,
     return instance;
 }
 
+std::unique_ptr<Instance> instantiate(const std::unique_ptr<Module>& module,
```

Comment:
This is not good idea, because it is too easy to copy Module unintentionally (as we have right now). If this is for avoid unittest modifications, move it `test/`.

Reply:
This is mostly for tests, but without this overload here it would be not clear why the only `instantiate` has `unique_ptr<Module>&&` parameter and not `unique_ptr<Module>` (the reason is rvalue-reference allows the call with lvalue `module` variable to be unambiguous)

Ok let me try the change suggested in https://github.com/wasmx/fizzy/pull/575#discussion_r500175696

Reply:
A comment explaining this would be enough to me. Later unit tests can be updated, when API is stable and we are happy with it.

Reply:
I moved second instantiate to `test/instantiate_helpers` and changed the main one to take `unique_ptr` by value, PTAL.